tesseract-server/src/schema_config.rs

```diff
@@ -15,7 +16,38 @@ pub fn read_schema(schema_path: &String) -> Result<Schema, Error> {
     } else if schema_path.ends_with("json") {
         schema = Schema::from_json(&schema_str)?;
     } else {
-        panic!("Schema format not supported");
+        return Err(format_err!("Schema format not supported"))
+    }
+
+    // Check each cube for unique level and property names
```

Comment:
Thanks for doing this check :) We should make sure to document this.

Reply:
Do we already have docs on writing schemas? I feel like that's where it should go, right?